product.js

```diff
@@ -0,0 +1,45 @@
+const products = [{
+        name: "Apple",
+        kcalper100g: "52"
+    },
+    {
+        name: "Egg",
+        kcalper100g: "140"
+    },
+    {
+        name: "Banana yoghurt",
+        kcalper100g: "86"
+    },
+    {
+        name: "Chicken and rice",
+        kcalper100g: "79"
+    },
+    {
+        name: "Orange",
+        kcalper100g: "79"
+    },
+
+];
+
+const productList = document.querySelector(".product-list");
+
+products.forEach(product => {
```

Comment:
Wyeliminuj ręcznie obszary, gdzie jest więcej niż jedna pusta linia, w weekend spróbuję zdebugować to zachowanie Prettiera.